register_forte_options.py

```diff
@@ -117,6 +118,15 @@ def register_mrcino_options(forte_options):
         "actice_docc and restricted_docc"
         "or not")
 
+def register_embedding_options(forte_options):
+        forte_options.add_bool("EMBEDDING", False, "Whether to perform embedding partition and projection")
+        forte_options.add_str("CUTOFF_BY", "THRESHOLD", "Cut off by: threshold or number.")
```

Comment:
The option doc is confusing. What number are we talking about? Number of MOs in the system fragment? Also, I agree with @lcyyork, please change this option to EMBEDDING_CUTOFF.